docs/topics/feed-exports.rst

```diff
@@ -268,6 +268,83 @@ in multiple files, with the specified maximum item count per file. That way, as
 soon as a file reaches the maximum item count, that file is delivered to the
 feed URI, allowing item delivery to start way before the end of the crawl.
 
+.. _post-processing:
+
+Post-Processing
+===============
+
+.. versionadded:: VERSION
+
+Scrapy provides an option to activate plugins to post-process items before
+they are exported to feed storages. In addition to using builtin plugins, users
+can create their own plugins. 
+
+These plugins can be activated through ``postprocessing`` :ref:`option of a feed <feed-options>`.
+Parameters to plugins can be passed through the feed settings where the key will be the parameter name
+and the value will be the parameter value. These parameters will be made accessible to the plugin
+through the ``feed_options`` parameter which will be passed to their constructor method.
+
+As the data is passed from one plugin to another, users must ensure the order of plugins do not
+cause a plugin to breakdown because of some unprocessable data.
+
+Each plugin is a Python class that must implement the following methods:
+
+.. method:: __init__(self, file, feed_options)
+
+    Constructor method to initialize plugin.
+
+    :param file: target file object to which post-processed data will be written
+
+    :param feed_options: feed-specific options
+    :type feed_options: :class:`dict`
+
+.. method:: write(self, data)
+
+   Process and write `data` to plugin's target file.
+
+   :param data: data passed to be written to file
+   :type data: :class:`bytes`
+
+.. method:: close(self)
+
+    Close the target file object.
+
+.. _builtin-plugins:
+
+Built-in Plugins
+----------------
+
+GzipPlugin
+^^^^^^^^^^
+
+Compresses recieved data with with :py:mod:`gzip` module.
+
+Accpeted parameters:
+
+-   ``gzip_compresslevel``: integer from 0 to 9 controlling the level of compression.
+-   ``gzip_mtime``: numeric timestamp for the compressed file.
+-   ``gzip_filename``: filename to be included in the header of compressed file.
```

Comment:
Use `.. autoclass::` instead of a section. That way you can move this documentation to the code, so that it’s written in a single place but can be read from both code and documentation. This also causes the header to be the whole import path of the class, which allows users to know which value to set in `postprocessing`.

About parameters, you should list them all since we don’t support every GZipFile option arbitrarily, but you don’t need to document anything about them separately. You can instead refer users to the documentation of `GZipFile` for information about them. That way you not only need to write less, you give users access to more information, and if a future version of Python changes how one of those parameters behaves, we won’t need to update our documentation.